lib/add_to_calendar.rb

```diff
@@ -27,6 +27,30 @@ def initialize(start_datetime:, end_datetime: nil, title:, timezone:, location:
   
       validate_attributes
     end
+    
+    def hey_url
+      calendar_url = "https://app.hey.com/calendar/ical_events/new?ical_source=BEGIN%3AVCALENDAR%0AVERSION%3A2.0%0APRODID%3A-%2F%2FBasecamp%2F%2FBC3%2F%2FEN%0ACALSCALE%3AGREGORIAN%0A%0ABEGIN%3AVEVENT"
```

Comment:
PRODID probably shouldn't be "Basecamp", but the product where this link is added to. It's Basecamp in the example link I sent you because it's a link extracted from Basecamp.